deeppavlov/models/syntax_parser/parser.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2019 Neural Networks and Deep Learning lab, MIPT
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import List
+import numpy as np
+
+from dependency_decoding import chu_liu_edmonds
+
+from deeppavlov.core.common.registry import register
+from deeppavlov.core.models.component import Component
+
+@register('chu_liu_edmonds_transformer') 
```

Comment:
```suggestion

@register('chu_liu_edmonds_transformer')
```

Reply:
Done.

Reply:
One more blank line before the class definition is needed, according to PEP8 codestyle https://www.python.org/dev/peps/pep-0008/#blank-lines.

Reply:
@AlexeySorokin

Reply:
Fixed, my apologies for being inattentive.